<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Network</class>
 <widget class="QDialog" name="Network">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>72</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>24</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Labyrinth</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>71</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QPushButton" name="host_button">
         <property name="text">
          <string>Host</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="hostPortEdit">
         <property name="placeholderText">
          <string>Enter Port</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="host_browser"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QPushButton" name="join_button">
         <property name="text">
          <string>Join</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="joinPortEdit">
         <property name="placeholderText">
          <string>Enter host port</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="joinIPEdit">
         <property name="placeholderText">
          <string>Enter Host IP</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="join_browser"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="startButton">
     <property name="text">
      <string>Start Game</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
